From ec286a0b613a9fa487be75b7c1c01e5c8ce62a1a Mon Sep 17 00:00:00 2001
From: Mark Hatle <mark.hatle@amd.com>
Date: Fri, 22 Sep 2023 11:01:16 -0600
Subject: [PATCH] meson.build: Microblaze does not support stack-protector

| cc1: warning: '-fstack-protector' not supported for this target
| ninja: build stopped: subcommand failed.

Upstream-Status: Inappropriate [Configuration]

Signed-off-by: Mark Hatle <mark.hatle@xilinx.com>
---
 meson.build | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/meson.build b/meson.build
index 395eca1943..8b87c5b3a2 100644
--- a/meson.build
+++ b/meson.build
@@ -405,14 +405,11 @@ possible_common_cc_flags = [
 
         '-fdiagnostics-show-option',
         '-fno-common',
-        '-fstack-protector',
-        '-fstack-protector-strong',
         '-fstrict-flex-arrays',
         '--param=ssp-buffer-size=4',
 ]
 
 possible_common_link_flags = [
-        '-fstack-protector',
 ]
 
 c_args = get_option('c_args')
-- 
2.34.1

